mango-core/src/main/scala/org/bdgenomics/mango/RDD/FrequencyRDD.scala

```diff
@@ -0,0 +1,124 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bdgenomics.mango.RDD
+
+import edu.berkeley.cs.amplab.spark.intervalrdd.IntervalRDD
+import org.apache.spark.Logging
+import org.apache.spark.rdd.RDD
+import org.apache.spark.storage.StorageLevel
+import org.bdgenomics.adam.models.ReferenceRegion
+import org.bdgenomics.formats.avro.AlignmentRecord
+
+import scala.collection.mutable.ListBuffer
+
+class FrequencyRDD extends Serializable with Logging {
+
+  var freqRDD: IntervalRDD[ReferenceRegion, SampleFreq] = null
+
+  /*
+   * Gets frequency data for samples over a given region.
+   *
+   * @param region: ReferenceRegion to query over
+   * @param samples: List[String of all samples to query for
+   *
+   * @return Map of (Sample, and list of FreqJson) at every location over the queried region
+   */
+  def get(region: ReferenceRegion, samples: List[String], sampleSize: Option[Int] = None): Map[String, Iterable[FreqJson]] = {
+    var data: RDD[(ReferenceRegion, SampleFreq)] = freqRDD.filterByInterval(region).toRDD
+    data = data.filter(r => samples.contains(r._2.sample))
```

Comment:
why not use IntervalRDD.filter? Was this too slow? We should investigate the performance tradeoff.